README.md

```diff
@@ -115,15 +115,19 @@ and scan files as they are uploaded to the S3 emulator.
 * If npm is not installed install [nvm](https://github.com/nvm-sh/nvm) in root directory.
 
 * Once nvm is installed:
-    `nvm install 13.6`
-
+    `nvm install 14.9`
```

Comment:
This should be fine, but can you remind people to upgrade their node versions once this PR gets merged?

Reply:
Sure, will do.